ersilia/hub/content/card.py

```diff
@@ -704,30 +706,33 @@ def get(self, model_id):
 class LocalCard(ErsiliaBase):
     def __init__(self, config_json):
         ErsiliaBase.__init__(self, config_json=config_json)
-
+    
+    @lru_cache(maxsize=32)
     def get(self, model_id):
-        model_path = self._model_path(model_id)
-        card_path = os.path.join(model_path, CARD_FILE)
+        model_path = os.path.join(EOS,  "dest", model_id)
```

Comment:
Why can't we use 'self._model_path?' here? Is there a particular reason for rewriting that to explicitly resolving the model path like this?